.eslintrc.json

```diff
@@ -115,6 +115,7 @@
 					"vs/nls",
 					"vs/css!./**/*",
 					"**/{vs,sql}/base/{common,browser}/**",
+					"**/{vs,sql}/platform/**/{common,browser}/**",
```

Comment:
Do not modify these - they're there to ensure that things are layered appropriately to avoid circular dependency issues.

Instead you should look at moving files as needed (and sometimes splitting things up in different files) if you're running into issues.